TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/autonomous/TimedSmartDuck.java

```diff
@@ -0,0 +1,103 @@
+package org.firstinspires.ftc.teamcode.opmode.autonomous;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DistanceSensor;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
+import org.firstinspires.ftc.teamcode.api.Robot;
+import org.firstinspires.ftc.teamcode.api.config.Naming;
+
+@Autonomous(name = "Timed Smart Duck", group = Naming.OPMODE_GROUP_COMP)
+public class TimedSmartDuck extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        Robot robot = new Robot(this);
+        DistanceSensor sensorL = hardwareMap.get(DistanceSensor.class, "distance_left");
+        DistanceSensor sensorR = hardwareMap.get(DistanceSensor.class, "distance_right");
+
+        telemetry.addData("Status", "Initialized");
+        telemetry.update();
+
+        waitForStart();
+
+        telemetry.addData("Status", "Running");
+        telemetry.update();
+
+        robot.closeClaw();
+
+        robot.armLeft.setPwmEnable();
+        robot.armRight.setPwmEnable();
+        robot.clawLeft.setPwmEnable();
+        robot.clawRight.setPwmEnable();
+
+        // Move left a little bit
+        ElapsedTime runtime = new ElapsedTime();
+
+        while (runtime.seconds() < 0.25 && opModeIsActive()) {
```

Comment:
For this whole file, you don't need the while loops. Just set the power to 0.5, sleep, then set the power to 0.

Reply:
Got it